### ISSUE TITLE
Large data

### ISSUE THREAD

**Issue by DeepKnowledge1** [xiahaifeng1995/PaDiM-Anomaly-Detection-Localization-master#2]
This work is impressive, 

I tried to use wide_resnet50_2 with the dataset(1k images), it worked fine, 
but when I increase the dataset to 2k or more, 

Do you have any idea how to run it with thousands of images

**Comment by xiahaifeng1995** [comment]
Does the training dataset with 2k images or the testing dataset? For the training dataset, I don't think it is necessary to extract so many positive sample features,  because the final size of the saved features is fixed. For the test set, I'm not sure if it's the cause of the memory overflow, maybe you can process the data in batches.

**Comment by DeepKnowledge1** [comment]
the max number of images was 1100, (size of 230*230), otherwise, I got the ram overflow,
